node/pkg/common/chainlock.go

```diff
@@ -250,5 +313,6 @@ func (msg *MessagePublication) ZapFields(fields ...zap.Field) []zap.Field {
 		zap.Uint8("consistency", msg.ConsistencyLevel),
 		zap.String("message_id", string(msg.MessageID())),
 		zap.Bool("unreliable", msg.Unreliable),
+		zap.String("verificationState", msg.verificationState.String()),
```

Comment:
Isn't this going to break the governor pending transfer DB stuff? Maybe this doesn't need to get marshalled and unmarshalled? Otherwise, you need to update `pkg/db/governor.go` to migrate from the old format to the new. 

Or maybe not, since that uses json marshal? Maybe you should at least have a test over there that verifies it can load an old `MessagePublication` from the DB into the new format?

Reply:
I'll look into it. I thought those worked on Transfer objects rather than MessagePublications. I'll look into adding those serialization tests though.

Reply:
Bruce and I discussed this more privately. Serializing this field is not necessary as it is currently not used by the Governor which is the only place that makes use of this DB and marshalling logic. 
I'll add a code comment noting this.